### PR TITLE
Finish java code challenge 3

### PR DESCRIPTION
**Pull request by PotatoGod123**
- [x ] Top-level README “Table of Contents” is updated
 - [x ] Feature tasks for this challenge are completed
 - x[ ] Unit tests written and passing
     - [ x] “Happy Path” - Expected outcome
     - [x ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [x ] README for this challenge is complete
     - [ x] Summary, Description, Approach & Efficiency, Solution
     - [x ] Link to code
     - [x ] Picture of whiteboard

### CODE REVIEW

File: challenges/test/lib/src/test/java/code/BinarySearchTest.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
+package code;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+public class BinarySearchTest {
+  @Test public void testBinarySearchFile(){
+    BinarySearch testMethod = new BinarySearch();
+    assertTrue(testMethod.testBinarySearch());
+  }
+
```
Comment (Mattpet26) [inline challenges/test/lib/src/test/java/code/BinarySearchTest.java:10]: Great work running multiple tests, don't forget to test for the base cases however! What if the array was empty or target value wasn't found!
